src/fetchMissing.js

```diff
@@ -1,12 +1,17 @@
-const fetch = require('node-fetch')
-const utils = require('web3-utils')
-
+const fetch = require('node-fetch');
+const utils = require('web3-utils');
+const fs = require('fs');
 async function getLogo(xdaiAddr) {
```

Comment:
I think we should rename this function because it returns a formatted address and not a logo